Vectors.Rmd

```diff
@@ -754,77 +754,74 @@ rownames(df3)
 
 *   Metadata is data, so storing it in a different way to the rest of the 
     data is fundamentally a bad idea. It also means that you need to learn
     a new set of tools to work with row names; you can't use what you already 
     know about manipulating columns.
  
-*   Row names are poor abstraction for labelling rows because they only work 
+*   Row names are a poor abstraction for labelling rows because they only work 
     when a row can be identified by a single string. This fails in many cases,
     for example when you want to identify a row by a non-character vector 
     (e.g. a time point), or with multiple vectors (e.g. position, encoded by
     latitude and longitude).
     
-*   Row names must be unique, so any replication of rows (e.g. from 
+*   Row names must be unique, so any duplication of rows (e.g. from 
     bootstrapping) will create new row names. If you want to match rows from 
-    before and after the transformation you'll need to perform complicated 
+    before and after the transformation, you'll need to perform complicated 
     string surgery.
 
     ```{r}
     df3[c(1, 1, 1), ]
     ```
 
-For these reasons, tibbles do not support row names. Instead the tibble package provides tools to easily convert row names into a regular column with either `rownames_to_column()`, or the `rownames` argument to `as_tibble()`:
+For these reasons, tibbles do not support row names. Instead the tibble package provides tools to easily convert row names into a regular column with either `rownames_to_column()`, or the `rownames` argument in `as_tibble()`:
 
 ```{r}
 as_tibble(df3, rownames = "name")
 ```
 
 ### Printing 
 
-One of the most obvious differences between tibbles and data frames is how they are printed. I assume that you're already familiar with how data frames are printed, so here I'll highlight some of the biggest differences using an example dataset included in the dplyr package:
+One of the most obvious differences between tibbles and data frames is how they print. I assume that you're already familiar with how data frames are printed, so here I'll highlight some of the biggest differences using an example dataset included in the dplyr package:
 
 ```{r}
 dplyr::starwars
 ```
 
-Tibbles:
-
-* Only show the first 10 rows and all the columns that will fit on screen.
-  Additional columns are shown at the bottom.
+* Tibbles only show the first 10 rows and all the columns that will fit on 
+  screen. Additional columns are shown at the bottom.
 
 * Each column is labelled with its type, abbreviated to three or four letters.
 
 * Wide columns are truncated to avoid a single long string occupying an entire
-  row. (This is still a work in progress: it's tricky to get the tradeoff right 
+  row. (This is still a work in progress: it's tricky to get the tradeoff 
   between showing as many columns as possible and showing a single wide 
-  column fully.)
+  column in its entirety just right.)
```

Comment:
This doesn't read very well to me (not the original was great either) — I had to re-read to understand what "just right" applied to

Reply:
"Wide columns are truncated to avoid having a single long string occupy an entire
row. (This is still a work in progress: it's a tricky tradeoff 
between showing as many columns as possible and showing columns in their entirety.)"

Reply:
:+1: